src/test/functional/functional-tests.js

```diff
@@ -117,7 +118,10 @@ describe('functional tests', function() {
   }
 
   before(done => client.makeBucket(bucketName, '', done))
-  after(done => client.removeBucket(bucketName, done))
+  after(done => {
+    client.removeObject(bucketName, objectName, () => {})
```

Comment:
where is it objectName getting created? because ideally it should be deleted where it gets created

also this style is not correct, you have to do:
```
client.removeObject(bucketName, objectName, (err) => {
    if (err) return done(err)
    client.removeBucket(bucketName, done)
})
```

or better, with promises:
```
client.removeObject(bucketName, objectName)
    .then(client.removeBucket(bucketName)
    .then(done)
    .catch(done)
```

Reply:
Removed it.